frontend/src/pages/FunctionalComponent/RecurringRunDetailsV2FC.tsx

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2023 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Regarding the name of the folder `frontend/src/pages/FunctionalComponent`, can we rename it to `frontend/src/pages/functional_components` to align with other folder naming pattern?